unitOneAssessmentPractice.js

```diff
@@ -4,39 +4,91 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const double = (num) => {
+  return num * 2
+}
+
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+    let str = num.toString()
+    let seven = "7"
+    if(str.includes(seven)){
```

Comment:
You can just `return str.includes("7")`, this is because includes already returns a boolean.

Reply:
Oooooh nice! Thank you!!! Good to know.